simulation/test_dht.cpp

```diff
@@ -143,7 +143,7 @@ TORRENT_TEST(dht_bootstrap)
 				ses.post_session_stats();
 				std::printf("depth: %d nodes: %d\n", routing_table_depth, num_nodes);
 				TEST_CHECK(routing_table_depth >= 8);
-				TEST_CHECK(num_nodes >= 110);
+				TEST_CHECK(num_nodes >= 50);
```

Comment:
I'm a little nervous about this. As far as I can tell the new code is causing a much smaller routing table at the end of bootstrap because if the local endpoint is a global address we will use that right away to set the node ID rather than doing an initial query with a random ID and changing it when we get the first response.